app/views/gigs/new.html.slim

```diff
@@ -0,0 +1,9 @@
+= provide :title, 'Add Gig'
+section.main-content.container
+  .row
+    .eight.columns.offset-by-two.columns
+      article
+        h2 Add Gig
+
+        = provide :button_name, 'Add gig'
```

Comment:
This is a mechanism to customize the button name by using a yield(:button_name) in the partial.  But to make it more explicit, I changed the code to pass locals instead.